text/0000-url-for.md

```diff
@@ -0,0 +1,47 @@
+- Start Date: 2015-09-12
+- RFC PR: (leave this empty)
+- Ember Issue: (leave this empty)
+
+# Summary
+
+This RFC proposes the extraction of a new `url-for` helper from `link-to`.
+
+# Motivation
+
+Decomposing `link-to` into a set of helpers will allow us to simplify its internals while providing new low level primitives for use in performance sensitive applications. `url-for`, the first of these extracted helpers, will allow for simple URL rendering in an Ember application. Extracting separate helpers for dealing with link classes will be suggested in a future RFC.
+
+This proposal would also allow plain URLs to be used within an Ember application without triggering an application reload. For example, a relative url such as `/about` could be used within a template or helper.
```

Comment:
Slight nit-pick: I personally wouldn't call this a relative url since it includes a leading slash.